scripts/rv_expression_association/README.md

```diff
@@ -0,0 +1,98 @@
+## Analysis plan
+
+The overall goal of these scripts is to run a gene-set association test to test for an association between i) a number of rare genetic variants located in and around a gene and ii) the expression level of the gene itself (in a given cell type).
+
+At first, I am testing specifically one chromosome 22 gene (_VPREB3_), in naive B cells.
```

Comment:
```suggestion
Testing is done on one gene within chromosome 22, VPREB3, in naive B cells, which will be further expanded to multiple genes.
```

Reply:
Out of curiosity, is this gene randomly-chosen?

Reply:
A mix! I had selected chromosome 22 somewhat at random (I was under the impression that this was the shortest chromosome, turns out that is actually chrom 21), but then since I was looking at expression in naive B cells I selected genes that were differentially expressed between naive B cells and all other cell types, hoping it would be an indication of a gene that is relevant for that specific cell type. That said at this stage this is largely just to start somewhere to build a working pipeline :)